router/clients.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const controller = require('../controllers/clients')
+
+const router = express.Router()
+
+const {
+  createClient,
+  updateClient,
+  deleteClient,
+  listClients
+} = controller
+
+router.post('/', createClient)
+router.put('/', updateClient)
+router.delete('/', deleteClient)
```

Comment:
you have to pass the id through the path in `updateClient` and `deleteClient` endpoints